app/controllers/api/v1/projects_controller.rb

```diff
@@ -36,7 +36,35 @@ def create
           end
         end
       end
+
+      def key
```

Comment:
would you mind renaming this to add_key or create_key. I feel like it would me slightly more clear